aten/src/ATen/native/ReflectionPad.cpp

```diff
@@ -479,7 +499,14 @@ Tensor& reflection_pad1d_out_cpu(
 }
 
 Tensor reflection_pad1d_cpu(const Tensor& input, IntArrayRef padding) {
-  auto output = at::empty({0}, input.options());
+  Tensor output;
+  if (input.is_quantized()) {
+    output = at::_empty_affine_quantized({0}, input.options(),
```

Comment:
this is assuming input is per tensor quantized, add a check for this?

Reply:
Good point -- I was only doing per_tensor. Will add an explicit check for it